utilities/doc/chIntroduction.tex

```diff
@@ -66,7 +66,7 @@ \section{Assumed Environment}\label{Sec:Env}
     \rowcolor{blue!30}
     \textbf{Component} & \textbf{reason} \\
     \hline
-    kerberos & to renew/check credentials via \texttt{klist} and \texttt{kinit} \\
+    kerberos & to renew/check credentials via \texttt{klist}, \texttt{kinit} and \texttt{aklog} \\
```

Comment:
I think `aklog` should be in the AFS line.
BTW: could you change:
`AFS (local mount)`
into:
`AFS (or openAFS for local mounting)`

Also:
`kerberos`
into:
`kerberos (local installation)`